app/src/main/java/io/multy/util/NetworkAvailability.java

```diff
@@ -11,6 +11,11 @@
 import android.net.ConnectivityManager;
 import android.net.NetworkInfo;
 
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+
 
 public class NetworkAvailability {
```

Comment:
name like NetworkUtils or NetworkStateHelper
availability sounds more likely for method name